include/boost/geometry/strategies/geographic/closest_points_cross_track_box_box.hpp

```diff
@@ -0,0 +1,303 @@
+#define BOOST_GEOMETRY_STRATEGIES_GEOGRAPHIC_CLOSEST_POINTS_CROSS_TRACK_BOX_BOX_HPP
+
+#include <boost/config.hpp>
+#include <boost/concept_check.hpp>
+#include <boost/mpl/if.hpp>
+#include <boost/type_traits/is_void.hpp>
+
+#include <boost/geometry/core/access.hpp>
+#include <boost/geometry/core/assert.hpp>
+#include <boost/geometry/core/point_type.hpp>
+#include <boost/geometry/core/radian_access.hpp>
+#include <boost/geometry/core/tags.hpp>
+
+#include <boost/geometry/algorithms/detail/closest_points/result.hpp>
+
+#include <boost/geometry/strategies/distance.hpp>
+#include <boost/geometry/strategies/concepts/distance_concept.hpp>
+#include <boost/geometry/strategies/spherical/distance_cross_track.hpp>
+#include <boost/geometry/strategies/geographic/distance_cross_track.hpp>
+#include <boost/geometry/strategies/spherical/distance_cross_track_box_box.hpp>
+#include <boost/geometry/strategies/spherical/closest_points_cross_track_box_box.hpp>
+
+#include <boost/geometry/util/math.hpp>
+#include <boost/geometry/algorithms/detail/assign_box_corners.hpp>
+
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace closest_points
+{
+
+
+template
+<
+    typename FormulaPolicy = strategy::andoyer,
+    typename Spheroid = srs::spheroid<double>,
+    typename CalculationType = void
+>
+class geographic_cross_track_box_box
+{
+public:
+
+    // point-point strategy getters
+    struct closest_points_pp_strategy
+    {
+        typedef geographic<FormulaPolicy, Spheroid, CalculationType> type;
+    };
+
+    // point-segment strategy getters
+    struct closest_points_ps_strategy
+    {
+        typedef geographic_cross_track
+                <
+                    FormulaPolicy,
+                    Spheroid,
+                    CalculationType
+                > type;
+    };
+
+    template <typename Box1, typename Box2>
+    struct calculation_type
+        : promote_floating_point
+          <
+              typename select_calculation_type
+                  <
+                      typename point_type<Box1>::type,
+                      typename point_type<Box2>::type,
+                      CalculationType
+                  >::type
+          >
+    {};
+
+    //constructor
+
+    explicit geographic_cross_track_box_box(Spheroid const& spheroid = Spheroid())
+             : m_spheroid(spheroid)
+    {}
+
+    template <typename Box1, typename Box2>
+    struct closest_point_result
+    {
+        typedef geometry::detail::closest_points::result
+        <
+            typename calculation_type<Box1, Box2>::type
+        > type;
+    };
+
+    template <typename Point, typename Box, typename Result>
+    static bool covered_by_box_with_info(Point const& box_corner,
+                                         Box const& other_box,
+                                         Result& result)
+    {
+        if (geometry::covered_by(box_corner, other_box))
```

Comment:
Here a geographic strategy should be called instead.

Reply:
both spherical and geographic strategies are using now `within::spherical_point_box`